popup.js

```diff
@@ -584,10 +587,25 @@ function clearTimes() {
   resetStartTimeChosen();
 }
 
+var EN_US = new Map();
+
+EN_US.set(400, 'Server said this request was invalid"')
+    .set(429, 'You have submitted too many sponsor times for this one video, are you sure there are this many?')
+    .set(409, 'This has already been submitted before')
+    .set(502, 'It seems the server is down. Contact the dev to inform them.')
+    .set('Unknown', 'here was an error submitting your sponsor times, please try again later.');
+
+function ErrorParser (Lang, statusCode) {
+  if(Lang.has(statusCode)) return Lang.get(statusCode);
+  Unknown = Lang.get('Unknown');
+  Unknown +=  Error code " + statusCode;
```

Comment:
Missing a quotation. Make sure to test if it builds before giving me the okay to merge.

Reply:
I did test just did not copy changes sorry